.github/workflows/offsets-presubmit.yaml

```diff
@@ -0,0 +1,27 @@
+name: verify-offsets
```

Comment:
Should this be added in its own PR?

Reply:
Yeah I'll break this into 2 PRs. I think the offsets are actually why we're seeing sudden failures like in #68. Looks like go1.20.3 was released yesterday and some of our checks are picking up the new version